src/components/Table/Table.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+import { Card } from '../card/Card'
+import { ITableStates } from "../TableInterfaces"
+import { IRedditData } from "../../RedditInterfaces"
+import './Table.css'
+
+interface ITableProps {
+    authorData: IRedditData['data']['children']
+}
+
+export const Table: React.FC<ITableProps> = ({ authorData }) => {
+
+    const [state, setState] = useState<ITableStates>({
```

Comment:
i think it is not appropriate to use names such as state and setState

Reply:
Ok I can rename.